### PR TITLE
adds a new phobia

### PR DESCRIPTION
**Pull request by kiwedespars**
## About The Pull Request

it's fucking everywhere and i hate it

image

## Why It's Good For The Game

idk you could asmite with it probably

## Changelog
:cl:
add: phobia of amogus
/:cl:

### CODE REVIEW

**Comment by LordSaladin** [comment]
amongphobia

**Comment by thakyZ** [comment]
I love to hate this and hate to love this

**Comment by TheEaterOfBeans** [comment]
merge or no balls (nuts, even)

File: strings/phobia.json [hunk 1]
```diff
@@ -341,5 +341,31 @@
...
+		"black",
+		"pink",
+		"cyan",
+		"white",
+		"emergency meeting",
+		"sus",
+		"vented",
+		"imposter",
+		"crewmate",
+		"electrical",
```
Comment (LetterN) [inline strings/phobia.json:364]: you forgot medbay

File: strings/phobia.json [hunk 2]
```diff
@@ -341,5 +341,31 @@
...
+		"yellow",
+		"purple",
+		"brown",
+		"black",
+		"pink",
+		"cyan",
+		"white",
+		"emergency meeting",
+		"sus",
+		"vented",
```
Comment (LetterN) [inline strings/phobia.json:361]: vent & vented

**Pull request closed without merging**